liiweb/templates/liiweb/locality_legislation_list.html

```diff
@@ -7,11 +7,16 @@
       <li class="breadcrumb-item">
         <a href="{% url 'locality_legislation' %}">{{ locality_legislation_title }}</a>
       </li>
+      <li class="breadcrumb-item active" aria-current="page">{{ locality.name }}</li>
     </ol>
   </nav>
 {% endblock %}
 {% block page-heading %}
-  <h1 class="my-4">{% blocktrans with locality=locality.name %}{{ locality }} Legislation{% endblocktrans %}</h1>
+  {% if locality.code in PROVINCIAL_CODES %}
```

Comment:
@nickmwangemi liiweb can't know about provinces :) this has to be overridden in lawlibrary, not here

Reply:
alternatively move the title into the view class and override it there.